public/service_portal/v0.1.0/openapi.json

```diff
@@ -428,6 +428,49 @@
                 }
             }
         },
+        "/portfolio_items/{portfolio_item_id}/icon": {
+            "get": {
+                "tags": [
+                    "users",
+                    "admins"
+                ],
+                "summary": "Fetches the specified portfolio item's icon information, reaching out to topology if necessary",
+                "operationId": "fetchServiceOfferingIconFromPortfolioItem",
+                "description": "Fetch the specified portfolio item's icon information.\n",
+                "parameters": [
+                    {
+                        "$ref": "#/components/parameters/PortfolioItemID"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Portfolio Item Icon",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "type": "array",
+                                    "items": {
+                                        "$ref": "#/components/schemas/ServiceOfferingIcon"
```

Comment:
Is items correct here? My understanding was a single object would be returned. Same question about the  `array` type.

Reply:
You're right, copy/paste error.